src/main/java/com/exasol/sql/ddl/create/rendering/CreateTableRenderer.java

```diff
@@ -126,28 +122,19 @@ public void visit(final IntervalYearToMonth intervalYearToMonthColumn) {
         append(getIntervalYearToMonthNameWithPrecision(intervalYearToMonthColumn));
     }
 
-    @Override
-    public void visit(final Field field) {
-        throw new UnsupportedOperationException();
-    }
-
     @Override
     public void visit(final Table table) {
         appendAutoQuoted(table.getName());
         setLastVisited(table);
     }
 
-    private String getIntervalDayToSecondNameWithPrecision(
-          final IntervalDayToSecond intervalDayToSecondColumn) {
-        return String.format(intervalDayToSecondColumn.getName(),
-              intervalDayToSecondColumn.getYearPrecision(),
+    private String getIntervalDayToSecondNameWithPrecision(final IntervalDayToSecond intervalDayToSecondColumn) {
```

Comment:
We really need to get the formatter running in your IDE.

Reply:
This class was created by me, so it's not a problem of eclipse formatting. I think line's length was 100 before, but then we decided to make it 120.